src/applications/mhv-medical-records/api/MrApi.js

```diff
@@ -217,22 +217,28 @@ export const getAllergies = async () => {
   });
 };
 
+export const getAllergiesWithOHData = async () => {
+  return apiRequest(
+    `${apiBasePath}/medical_records/allergies?use_oh_data_path=1`,
+    {
+      headers,
+    },
+  );
+};
+
 export const getAllergy = id => {
   return apiRequest(`${apiBasePath}/medical_records/allergies/${id}`, {
     headers,
   });
 };
 
 export const getAcceleratedAllergies = async () => {
-  return apiRequest(
-    `${apiBasePath}/medical_records/allergies?use_oh_data_path=1`,
-    {
-      headers,
-    },
-  );
+  return apiRequest(`${API_BASE_PATH_V2}/medical_records/allergies`, {
+    headers,
+  });
 };
```

Comment:
Maybe we should not access the SCDF endpoint `${API_BASE_PATH_V2}/medical_records/allergies` until the next PR?
Otherwise it may break Production which currently has feature flag `mhv_accelerated_delivery_allergies_enabled` set to true.